compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -169,7 +169,7 @@ void update_quantparam(record_minmax::MinMaxObserver *observer, const std::strin
 namespace record_minmax
 {
 
-void RecordMinMax::initialize(const std::string &input_model_path)
+void RecordMinMax::initializeCircleModule(const std::string &input_model_path)
```

Comment:
can you please make this diff better to review?
unintended changes are visible and I cannot catch the changes.

Reply:
Done